k8s/tools/deploy-pubsub-gke.sh

```diff
@@ -155,17 +159,23 @@ if [[ "$*" != *--no-cloudfunctions* ]] ; then
   done
 fi
 
+# Enable App Engine
```

Comment:
Does Turbinia use app engine for something?

Reply:
Not directly, but App Engine is required to create a Datastore database.